Makefile

```
@@ -134,12 +134,12 @@ install: $(KUSTOMIZE) manifests ## Install CRDs into a cluster
 
 .PHONY: uninstall
 uninstall: $(KUSTOMIZE) manifests ## Uninstall CRDs from a cluster
-	kustomize build config/crd | kubectl delete -f -
+	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 .PHONY: deploy
 deploy: $(KUSTOMIZE) manifests ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-	cd config/manager && kustomize edit set image controller=${IMG}
-	kustomize build config/default | kubectl apply -f -
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/default &> config/render/kustomization.yaml
```

```suggestion
	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
	$(KUSTOMIZE) build config/default | kubectl apply -f -
```
You need to add `$(KUSTOMIZE) build config/default &> config/render/kustomization.yaml` on the next target, `manifests` and add a dependency to `$(KUSTOMIZE)` there too.
The name kustomization.yaml might be misleading as one would expect a kustomization. Maybe deployment.yaml or something similar ?